templates/airflow-ec2.j2

```diff
@@ -0,0 +1,338 @@
+              mode: '000400'
+              owner: root
+              group: root
+            /lib/systemd/system/cfn-hup.service:
+              content: !Sub |
+                [Unit]
+                Description=cfn-hup daemon
+
+                [Service]
+                Type=simple
+                ExecStart=/opt/aws/bin/cfn-hup
+                Restart=always
+
+                [Install]
+                WantedBy=multi-user.target
+              mode: '000400'
+              owner: root
+              group: root
+          commands:
+            01_enable_cfn-hup:
+              command: '/bin/systemctl enable cfn-hup.service'
+            02_start_cfn-hup:
+              command: '/bin/systemctl start cfn-hup.service'
+        install_tools:
+          commands:
+            01_jq:
+              command: '/usr/bin/apt install -y jq'
+            02_curl:
+              command: '/usr/bin/apt install -y curl'
+            03_vim:
+              command: '/usr/bin/apt install vim -y'
+            04_less:
+              command: '/usr/bin/apt install less -y'
+        config_airflow_service:
+          files:
+            '/etc/supervisor/supervisord.conf':
+              mode: '000644'
+              owner: 'root'
+              group: 'root'
+              content: |
+                ; supervisor config file
+
+                [unix_http_server]
+                file=/var/run/supervisor.sock   ; the path to the socket file
+                chmod=0700                      ; sockef file mode (default 0700)
+
+                [supervisord]
+                loglevel=debug                              ; log level; default info; others: debug,warn,trace
+                logfile=/var/log/supervisor/supervisord.log ; (main log file;default $CWD/supervisord.log)
+                pidfile=/var/run/supervisord.pid            ; (supervisord pidfile;default supervisord.pid)
+                childlogdir=/var/log/supervisor             ; ('AUTO' child log dir, default $TEMP)
+
+                ; the below section must remain in the config file for RPC
+                ; (supervisorctl/web interface) to work, additional interfaces may be
+                ; added by defining them in separate rpcinterface: sections
+                [rpcinterface:supervisor]
+                supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+                [supervisorctl]
+                serverurl=unix:///var/run/supervisor.sock ; use a unix:// URL  for a unix socket
+
+                ; create a group for all airflow-related programs
+                ; to issue commands that pertain to the whole group,
+                ; use a wildcard, for example, `supervisorctl start airflow:*`
+                [group:airflow]
+                programs=airflow
+
+                [program:airflow]
+                command=docker compose -f /root/orca-recipes-airflow/docker-compose.yaml up --build
```

Comment:
Do you set up the new password after initiation?

Reply:
Yes, a future bit of work is to figure out how to configure the admin account password using the secrets backend.

Reply:
To do this we could add two more steps to the cloudformation template.

1. get the secret "ariflow_password" from secrets manager
2. Create a new admin user with our password using the [airflow cli](https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#users)

There might be some more installing or configuring necessary to make this happen

Reply:
I will create a ticket for us to track that.